test/e2e/service.go

```diff
@@ -548,6 +548,16 @@ var _ = Describe("Services", func() {
 			i++
 		}
 
+		By("updating service's port " + serviceName + " and reaching it at the same IP")
+		service, err = updateService(f.Client, f.Namespace.Name, serviceName, func(s *api.Service) {
+			s.Spec.Ports[0].Port = 19482
```

Comment:
The original port used is 80, and is hard-coded all over the place throughout this file. It originally comes from the BuildServiceSpec function.

I was originally going to make the port here the original port+1, but using a reserved port for something it's not intended for felt dirty